caravel/assets/javascripts/SqlLab/components/QueryLog.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as Actions from '../actions';
+
+import QueryTable from './QueryTable';
+import { Alert } from 'react-bootstrap';
+
+class QueryLog extends React.Component {
```

Comment:
since this component is connection to stores, i might call it `QueryLogContainer`